internal/graph/schema/service_maintenance.graphqls

```diff
@@ -0,0 +1,41 @@
+type RunMaintenancePayload {
+	error: String
+}
+
+input RunMaintenanceInput {
+	project: String!
+	serviceName: String!
+	teamSlug: Slug!
+	environmentName: String!
+}
+
+extend type Mutation {
+	RunMaintenance(input: RunMaintenanceInput!): RunMaintenancePayload
+}
+
+extend type ValkeyInstance {
+	"Fetch maintenances for the Valkey instance."
+	maintenance: ServiceMaintenance!
+	project: String!
+}
+
+type ServiceMaintenance {
+	updates: [ServiceMaintenanceUpdate]!
+}
+
+type ServiceMaintenanceUpdate {
```

Comment:
Denne typen virker også som at potensielt er veldig generisk. Feltene tillater det, men er det riktig at den skal brukes for all maintenance?